frontend/src/data/nces322_50/categories.ts

```diff
@@ -0,0 +1,55 @@
+export const Categories = {
+    'race': new Set<string>([
+        "WhiteAlone",
+        "Black",
+        "HispanicOrLatino",
+        "AsianOrPacificIslander",
+        "Asian",
+        "HawaiianNativeOrPacificIslander",
+        "AmericanIndianOrAlaskaNative",
+        "TwoOrMoreRaces",
+    ]),
+    'bachelorsDegreeMajor': new Set<string>([
+        "BachelorOfAgricultureAndNaturalResourcesMajor",
+        "BachelorOfArchitectureAndRelatedServicesMajor",
+        "BachelorOfEthnicCulturalGenderGroupStudiesMajor",
+        "BachelorOfBiologicalAndBiomedicalSciencesMajor",
+        "BachelorOfBusinessMajor",
+        "BachelorOfCommunicationJournalismAndRelatedMajor",
+        "BachelorOfCommunicationsTechnologiesMajor",
+        "BachelorOfComputerAndITMajor",
+        "BachelorOfEducationMajor",
+        "BachelorOfEngineeringMajor",
+        "BachelorOfEngineeringTechnologiesMajor",
+        "BachelorOfEnglishAndLiteratureMajor",
+        "BachelorOfFamilyAndConsumerSciencesMajor",
+        "BachelorOfForeignLanguagesAndLinguisticsMajor",
+        "BachelorOfHealthRelatedMajor",
+        "BachelorOfHomelandSecurityLawEnforcementFirefightingMajor",
+        "BachelorOfLegalProfessionsAndStudiesMajor",
+        "BachelorOfArtsHumanitiesAndOtherMajor",
+        "BachelorOfLibraryScienceMajor",
+        "BachelorOfMathAndStatisticsMajor",
+        "BachelorOfMilitaryTechnologiesMajor",
+        "BachelorOfInterdisciplinaryStudiesMajor",
+        "BachelorOfParksRecFitnessKinesiologyMajor",
+        "BachelorOfPhilosophyReligiousStudiesMajor",
+        "BachelorOfPhysicalSciencesAndTechnologiesMajor",
+        "BachelorOfPrecisionProductionMajor",
+        "BachelorOfPsychologyMajor",
+        "BachelorOfPublicAdminSocialServicesMajor",
+        "BachelorOfSocialSciencesMajor",
+        "BachelorOfHistoryMajor",
+        "BachelorOfTheologyAndReligionMajor",
+        "BachelorOfTransportationAndMaterialsMovingMajor",
+        "BachelorOfVisualAndPerformingArtsMajor",
+        "BachelorOfOtherMajor",
+    ]),
+    'residentStatus': new Set<string>([
```

Comment:
The nonusresident dimension is in the same category location as race. If it's included in the total for summary statistics that include each dimenion of the race category, we can move it to the race category.

Reply:
done